ui/src/services/api.js

```diff
@@ -129,11 +129,10 @@ export async function deployNode(url, token, node) {
       {
         client: 'runner',
         fun: 'state.orchestrate',
-        arg: ['metalk8s.orchestrate.deploy_new_node'],
+        arg: ['metalk8s.orchestrate.deploy_node'],
         kwarg: {
           saltenv: 'metalk8s-2.0',
```

Comment:
@carlito-scality @MonPote Is there a tracking ticket to get rid of this hard-coded value?

Reply:
@NicolasT See https://github.com/scality/metalk8s/issues/1105